sdk/sdktypes/value_wrap.go

```diff
@@ -83,11 +83,13 @@ func (w ValueWrapper) Wrap(v any) (Value, error) {
 		fs := make(map[string]Value)
 
 		for _, vfs := range reflect.VisibleFields(vt) {
-			if !vfs.IsExported() || vfs.Anonymous {
-				// allow to wrap embedded unexported time.Time field
-				if vfs.Type != reflect.TypeOf(time.Time{}) {
-					continue
-				}
+			if !vfs.IsExported() {
+				continue
+			}
+
+			if vfs.Anonymous && vfs.Type != reflect.TypeOf(time.Time{}) {
```

Comment:
This seems very specific, can you document why we look only at `time.Time`?

Reply:
added a note